Mathlib/Tactic/Linter/MinImports.lean

```diff
@@ -0,0 +1,109 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import ImportGraph.Imports
+import Mathlib.Tactic.MinImports
+
+/-! # The `minImports` linter
+
+The `minImports` linter incrementally computes the minimal imports needed for each file to build.
+Whenever it detects that a new command requires an increase in the (transitive) imports that it
+computed so far, it emits a warning mentioning the bigger minimal imports.
+
+Unlike the related `#min_imports` command, the linter takes into account notation and tactic
+information.
+It also works incrementally, providing information that it better suited, for instance, to split
```

Comment:
This sentence has a grammar mistake around "that it better suited", but I'm not sure in which way to fix it.

Reply:
Corrected the mistake and explained a little better.